test/foundry-tests/utils/MockInvalidERC20.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import {ERC20} from "solmate/tokens/ERC20.sol";
+
+contract MockInvalidERC20 is ERC20 {
```

Comment:
We can just use `TestInvalidERC20` which is in the top level `contracts/test` folder right?

Reply:
Right, fixed it!